src/test/scala/com/fluidcode/processing/silver/GetPostsInfoAndCreateTableSpec.scala

```diff
@@ -0,0 +1,82 @@
+package com.fluidcode.processing.silver
+
+import com.fluidcode.configuration.Configuration
+import com.fluidcode.models.bronze.{Captions, Comments, CommentsData, Data, DataElements, DimensionsData, EdgesElements,
+                            GraphImagesElements, Info, Likes, NodeData, Owner, OwnerData, ProfileInfo, ThumbnailElements}
```

Comment:
try with "._" instead if naming all case classes in the import

Reply:
Ok!